### DISCUSSION TITLE
Add precision date fields in Accession form

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5970] [Feature Requests]
**Is your feature request related to a problem? Please describe.**

> As mentioned in the following post: [Clarification on Date precission fields and their use 3](https://discourse.specifysoftware.org/t/clarification-on-date-precission-fields-and-their-use/1813), it would be necessary if the calendar date fields in the Accession form had the active date selector, fields like date of accession or date of reception. Like many museums, we have historical collections with incomplete dates for some specimens.

This request essentially seeks to add precision date fields for 
* `DateAccessioned`
* `DateAcknowledged`
* `DateReceived`

These are currently just calendar fields without the ability to specify precision. To remedy this, we need to add:

* `DateAccessionedPrecision` (byte field)
* `DateAcknowledgedPrecision` (byte field)
* `DateReceivedPrecision` (byte field)

**Reported By**
Jordi at MCNB
Nate Shoobs at OSU

**Additional context**

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/add-precision-data-fields-in-accession-form/1851/3

**Comment by grantfitzsimmons** [comment]
Fedor is also in favor of this being added

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8177
